.github/workflows/publish.yml

```
@@ -0,0 +1,38 @@
+name: publish
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
```

Is "3.x" a valid version?

---

```suggestion
          python-version: '3.10'
```

3.x works on my other projects, but you're right I think it's better to specify here so that it doesn't pull unsupported versions like 3.8.